docs/local-testing-with-crc.md

```diff
@@ -0,0 +1,39 @@
+# Local testing with CRC
+
+We can use CRC as an Openshift-like environment within which to test downstream OLM. [CRC](https://developers.redhat.com/products/codeready-containers/overview) 
+is a tool for deploying a local Openshift cluster on your laptop.
+
+TL;DR
+
+1. Install [CRC](https://developers.redhat.com/products/codeready-containers/overview)
+2. `make crc` to provision a CRC cluster, build OLM, and deploy it on the cluster
+3. `make e2e WHAT=operator-lifecycle-manager E2E_INSTALL_NS=openshift-operator-lifecycle-manager E2E_TEST_NS=openshift-operators`, i.e., execute e2e tests as you normally would.
```

Comment:
This command looks like a mirror of what `make e2e/olm` is doing under-the-hood. It just shells out to OLM's `make e2e` command and overrides the install/test namespace variables to point to the downstream ones.

Reply:
Nice, fixed!